apis/sgv2-quarkus-common/src/main/java/io/stargate/sgv2/api/common/cql/ExpressionUtils.java

```diff
@@ -0,0 +1,33 @@
+package io.stargate.sgv2.api.common.cql;
+
+import com.bpodgursky.jbool_expressions.And;
+import com.bpodgursky.jbool_expressions.Expression;
+import com.bpodgursky.jbool_expressions.Or;
+import java.util.List;
+
+public class ExpressionUtils<K> {
+
+  public static <K> And<K> OrderedAndOf(Expression<K>... expressions) {
```

Comment:
On naming: should follow Java naming convention, so even if leaving "ordered" in there, should be

    public static .... orderedAndOf(...)

(that is, start with lower-case letter)

Reply:
done